pkg/diskmaker/diskmaker.go

```diff
@@ -55,7 +62,25 @@ func (d *DiskMaker) loadConfig() (DiskConfig, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error unmarshalling %s with %v", d.configLocation, err)
 	}
-	return diskConfig, nil
+
+	lv := &localv1.LocalVolume{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      diskConfig.OwnerName,
+			Namespace: diskConfig.OwnerNamespace,
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind:       diskConfig.OwnerKind,
+			APIVersion: diskConfig.OwnerAPIVersion,
+		},
+	}
+	lv, err = d.apiClient.getLocalVolume(lv)
+
+	if err != nil {
+		return nil, fmt.Errorf("error fetching local volume %s/%s with %v", diskConfig.OwnerName, diskConfig.OwnerNamespace, err)
```

Comment:
nit: `error fetching local volume %s/%s: %v`
nit: use `namespace / name`, not the other way around